prettyconf/casts.py

```diff
@@ -93,3 +95,11 @@ def __call__(self, value):
             return self.options[value]
         except KeyError:
             raise InvalidConfiguration("Invalid option {!r}".format(value))
+
+
+class JSON(AbstractCast):
+    def __call__(self, value):
+        try:
+            return json.loads(value)
+        except json.JSONDecodeError:
+            raise InvalidConfiguration('Invalid option {!r}'.format(value))
```

Comment:
Isn't it better to make a raise x from e so you keep track of the original exception?